types/anybar/index.d.ts

```diff
@@ -5,6 +5,4 @@
 
 declare namespace anybar { }
 
-declare function anybar(color: string, options?: { port: number }): void
-
-export = anybar
+export default function anybar(color: string, options?: { port: number }): void
```

Comment:
This isn't correct, [it's definitely an `export =` declaration](https://github.com/sindresorhus/anybar/blob/8371af96bcb6ff3dee5af6b4b4f02153f4b84eb7/index.js#L5) and should be imported with `import anybar = require("anybar")` or as `import anybar from 'anybar'` by users who want to use 2.7's `esModuleInterop` or 1.8's `allowSyntheticDefaultImports`. [Read more here](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/df0a8726871810b6bab0ae6cc1ce5d1ea58b827d#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default).

Could you instead remove the empty `declare namespace anybar {}`, fix up the version to `2.0` and enable whatever lint rules are disabled?

Reply:
Why does `export function anybar` not work while using the  `declare function anybar ... export = anybar` approach it does? The first one results in `Cannot invoke an expression whose type lacks a call signature`. I'm not experienced enough to understand why, maybe you can clear this up for me? :)

Reply:
`export function anybar` gets emitted closer to something like